entity-framework/core/providers/index.md

```diff
@@ -1,17 +1,75 @@
+| [Devart.Data.SQLite.EFCore](https://www.nuget.org/packages/Devart.Data.SQLite.EFCore/)                            | SQLite 3 onwards           | [DevArt](https://www.devart.com/)                                             | Paid                           | [docs](https://www.devart.com/dotconnect/sqlite/docs/)                                                                                                                    |
+| [Devart.Data.MySql.EFCore](https://www.nuget.org/packages/Devart.Data.MySql.EFCore/)                              | MySQL 5 onwards            | [DevArt](https://www.devart.com/)                                             | Paid                           | [docs](https://www.devart.com/dotconnect/mysql/docs/)                                                                                                                     |
+| [EntityFrameworkCore.Jet](https://www.nuget.org/packages/EntityFrameworkCore.Jet/)                                | Microsoft Access files     | [Bubi](https://github.com/bubibubi)                                           | EF Core 2.0, .NET Framework    | [readme](https://github.com/bubibubi/EntityFrameworkCore.Jet/blob/master/docs/README.md)                                                                                  |
+
+## Future Providers
+
+### Cosmos DB
+
+We have been developing an EF Core provider for the DocumentDB API in Cosmos DB. This will be the first complete document-oriented database provider we have produced, and the learnings from this exercise are going to inform improvements in the design of the subsequent release after 2.1. The current plan is to publish an early preview of the provider in the 2.1 timeframe.
+
+### Oracle
+The Oracle .NET team has announced they are planning to release a first-party provider for EF Core 2.0 approximately in the third quarter of 2018. See their [statement of direction for .NET Core and Entity Framework Core](http://www.oracle.com/technetwork/topics/dotnet/tech-info/odpnet-dotnet-ef-core-sod-4395108.pdf) for more information.
+Please direct any questions about this provider, including the release timeline, to the [Oracle Community Site](https://community.oracle.com/).
+
+In the meanwhile, the EF team has produced a [sample EF Core provider for Oracle databases](https://github.com/aspnet/EntityFrameworkCore/blob/dev/samples/OracleProvider/README.md). The purpose of the project is not to produce an EF Core provider owned by Microsoft, but to help us identify gaps in EF Core's relational and base functionality which we need to address in order to better support Oracle, and to jumpstart the development of other Oracle providers for EF Core either by Oracle or third parties.    
```

Comment:
Maybe add "We would welcome and encourage a community effort to create an open-source Oracle provider for EF Core, using the sample as a starting point."